gtests/net/mptcp/dss/dss_fin_retrans_close_wait.pkt

```diff
@@ -29,4 +29,5 @@
 
 // ACK the data_fin
 +0       <   .  2:2(0)  ack 1  win 450    <dss dack4=2 nocs>
-+0       >  R.  1:1(0)  ack 2             <mp_fastclose, mp_reset 0>
++0       >  F.  1:1(0)  ack 2             <dss dack4=2 nocs>
++0       <   .  2:2(0)  ack 2  win 450    <dss dack4=2 nocs>
```

Comment:
(detail: for packetdrill, finishing with an injected packet doesn't have any influence: the packet is going to be injected but Packetdrill is not going to check anything else after that → I don't see what else you want to check here anyway :) )

But it doesn't hurt: it can still be used to check if this doesn't create a kernel crash :)

(same in `gtests/net/mptcp/dss/dss_fin_retrans_established.pkt` but there we could check if the last ACK is sent?)